controllers/project_mesh_controller.go

```diff
@@ -19,6 +19,7 @@ import (
 const (
 	AnnotationServiceMesh = "opendatahub.io/service-mesh"
 	AnnotationGatewayHost = "opendatahub.io/service-mesh-gw-host"
+	AnnotationGateway     = "opendatahub.io/service-mesh-gateway"
```

Comment:
Should we consider just "gw" for the sake of alignment with AnnotationGatewayHost?

Reply:
```suggestion
	AnnotationGateway     = "opendatahub.io/service-mesh-gw"
```